app/views/teams/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for :main_content do %>
     <% javascript 'teams.js' %>
-    <div class="panel panel-info" style="width: min-content">
```

Comment:
Alternatively I could change the `width` attribute to `max-content` and let the size of the buttons remain as they were